content/attacks/posts/2017-04-22-Yapizon.md

```diff
@@ -0,0 +1,41 @@
+---
+date: 2017-04-22
+target-entities: Yapizon
+entity-types:
+- Exchange
+- Custodian
+attack-types:
+- Wallet Hack
+tags:
+- North Korea
+- Lazarus Group
+title: "Yapizon suffers over $5.5 million in losses from a hacker attack."
+---
+
+## Summary
+
+On April 22, 2017, Yapizon, a South Korean cryptocurrency exchange,[suffered a security breach](https://www.newsbtc.com/news/south-korean-yapizon-bitcoin-exchange-confirms-hack-loses-5-mil-bitcoin/) in which hackers gained unauthorized access to the exchange's hot wallets, resulting in the theft of approximately $5.5 million worth of cryptocurrency. Later, after this incident, the exchange rebranded as Youbit.
+
+## Attackers
+
+[Lazarus Group](https://cointelegraph.com/news/south-korean-bitcoin-exchange-yapizon-starts-trading-bitcoin-cash), a North Korean hacker crew has been accused of carrying out the heist. That was one of the first cryptocurrency-related hacks that Lazarus Group has been blamed for.
+
+## Losses
+
+The total losses are 3816 BTC, equivalent to around $5.5 million [(about 37% of Yapizon total assets)](https://mrhacker.co/news/south-korean-bitcoin-exchange-yapizon-hacked-5-million-stolen)].
+
+## Timeline
+
+- **April 22, 2017:** Yapizon [had funds drained from its hot wallets](https://www.bleepingcomputer.com/news/security/hacked-south-korean-bitcoin-exchange-loses-5-5-million/).
+- **April 25, 2017:** Yapizon [announces the hack and decides to spread the losses to all user accounts](https://archive.is/97ZjO).
+- **April 26, 2017:** Yapizon [has suspended operations for the next few days](https://www.cryptoninjas.net/2017/04/26/south-korean-bitcoin-exchange-yapizon-hacked-3831-btc/) in order to finish accounting work and strengthen security systems.
+
+
+## Security Failure Causes
+
+Yapizon has not officially announced what exactly allowed the hackers to access its system. Several possible reasons:
```

Comment:
dont speculate. if there are credible sources that discuss reasons for the hack, you can mention/link them.  otherwise just leave as unknown.